web/src/util.ts

```diff
@@ -281,10 +281,17 @@ export function is_stream_synonym(text: string): boolean {
     return text === "channel";
 }
 
-export function canonicalize_stream_synonym(text: string): string {
+export function is_streams_synonym(text: string): boolean {
+    return text === "channels";
+}
+
+export function canonicalize_stream_synonyms(text: string): string {
```

Comment:
When this is called either only one of the two possible strings will ever be passed to it or either one will and therefore the correct singular/plural of "stream" needs to be returned.

Reply:
This function probably deserves a comment noting its purpose -- for parts of the codebase that have been converted to use `channel/channels` internally, this is used to convert those back into `stream/streams` for external presentation.

Reply:
Added this comment to the function in the updated changes.